src/MoveUntilTouchTopicController.cpp

```diff
@@ -147,93 +90,7 @@ bool MoveUntilTouchTopicController::shouldAcceptRequests() {
 
 //=============================================================================
 bool MoveUntilTouchTopicController::shouldStopExecution(std::string &message) {
-  // inelegent to just terminate any running trajectory,
-  // but we must guarantee taring completes before starting
-  if (!mTaringCompleted.load()) {
-    ROS_WARN("taring not yet completed!");
-    return true;
-  }
-
-  double forceThreshold = mForceThreshold.load();
-  double torqueThreshold = mTorqueThreshold.load();
-
-  std::lock_guard<std::mutex> lock(mForceTorqueDataMutex);
-  bool forceThresholdExceeded = mForce.norm() >= forceThreshold;
-  bool torqueThresholdExceeded = mTorque.norm() >= torqueThreshold;
-
-  if (forceThresholdExceeded) {
-    std::stringstream messageStream;
-    messageStream << "Force Threshold exceeded!   Threshold: " << forceThreshold
-                  << "   Force: " << mForce.x() << ", " << mForce.y() << ", "
-                  << mForce.z();
-    message = messageStream.str();
-    ROS_WARN(message.c_str());
-  }
-  if (torqueThresholdExceeded) {
-    std::stringstream messageStream;
-    messageStream << "Torque Threshold exceeded!   Threshold: "
-                  << torqueThreshold << "   Torque: " << mTorque.x() << ", "
-                  << mTorque.y() << ", " << mTorque.z();
-    message = messageStream.str();
-    ROS_WARN(message.c_str());
-  }
-
-  return forceThresholdExceeded || torqueThresholdExceeded;
-}
-
-//=============================================================================
-void MoveUntilTouchTopicController::setForceTorqueThreshold(
-    FTThresholdGoalHandle gh) {
-  const auto goal = gh.getGoal();
-  ROS_INFO_STREAM("Setting thresholds: force = " << goal->force_threshold
-                                                 << ", torque = "
-                                                 << goal->torque_threshold);
-  FTThresholdResult result;
-  result.success = true;
-  if (!isRunning()) {
```

Comment:
Do we need to keep this logic somewhere in the new code?

Reply:
Made it explicit by turning the Action Server on and off with the controller.